pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -272,8 +273,8 @@ async def run(
                     else:
                         # continue the conversation
                         handle_span.set_attribute('tool_responses', tool_responses)
-                        tool_responses = ' '.join(r.part_kind for r in tool_responses)
-                        handle_span.message = f'handle model response -> {tool_responses}'
+                        tool_responses_str = ' '.join(r.part_kind for r in tool_responses)
+                        handle_span.message = f'handle model response -> {tool_responses_str}'
```

Comment:
What's the point of this year? Maybe I'm not quite understanding...

Wouldn't it result in:

`handle model response -> text_part text_part tool_part` or something like that?

Reply:
well they're `ModelRequestPart`s, but yes.

It could be improved, but this is good for now.